TeamHyperAlphaWolfDarkZero/src/GUI/EmployeeGUI.java

```diff
@@ -14,18 +15,18 @@
 import javax.swing.JButton;
 
 public class EmployeeGUI extends JPanel {
-	private JTextField text_id;
-	private JTextField text_firstName;
-	private JTextField text_lastName;
-	private JTextField text_Address;
-	private JTextField text_postcode;
-	private JTextField text_City;
-	private JTextField text_NationInsurance;
-	private JTextField text_AccountName;
-	private JTextField text_bic;
-	private JTextField text_iban;
-	private JTextField text_startSalary;
-	private JTextField text_department;
+	public JTextField text_id;
+	public JTextField text_firstName;
+	public JTextField text_lastName;
+	public JTextField text_Address;
+	public JTextField text_postcode;
+	public JTextField text_City;
+	public JTextField text_NationInsurance;
+	public JTextField text_AccountName;
+	public JTextField text_bic;
+	public JTextField text_iban;
+	public JTextField text_startSalary;
+	public JTextField text_department;
```

Comment:
Just to query why these fields were these changed from private to public? Was the time constraint an issue?